playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
 
 EDXAPP_SESSION_COOKIE_SECURE: false
 
+EDXAPP_INSTRUCTOR_QUERY_PROBLEM_TYPES:
+  - 'problem',
+  - 'openassessment',
```

Comment:
Where did we get this list from?

This looks like old ORA1, which we are no longer using.